app/incident/delete/template.hbs

```diff
@@ -4,6 +4,6 @@
     updateButtonAction=updateButtonAction
     updateButtonText=updateButtonText}}
   <div class="alert alert-danger">
-    <span class="glyphicon glyphicon-warning-sign"></span> {{t 'incident.messages.deleteIncident' name=name}}
+    <span class="glyphicon glyphicon-warning-sign"></span> {{t 'messages.delete_singular' name='incident'}}
```

Comment:
This code is passing a non localized string when it should be passing in a localized string or it should use the name of the item being deleted.